lib/assessment.js

```diff
@@ -5,7 +5,16 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
-
+  var arg = arguments;
+  if(arg){
+    var minimum = arg[0];
+    // this for loop is not working, and I'm not sure why. It's not entering the for loop.
+    for(var i = 0, length = arg.lenth; i < length; i++){
+      if(arg[i] < minimum){
+        minimum = arg[i];
+      }
+    } return minimum;
+  } else return console.log("What is the minimm of nothing?");
```

Comment:
I tested your code, and (with a few minor syntactical corrections) it seems to work. See below:
```javascript
var min = function min() {
    var arg = arguments;
    if(arg){
        var minimum = arg[0];
        for(var i = 0, length = arg.lenth; i < length; i++){
          if(arg[i] < minimum){
            minimum = arg[i];
          }
        } 
        return minimum;
    } else {
        return console.log("What is the minimm of nothing?");
    }	 
 };
```